R/sql-context.R

```diff
@@ -0,0 +1,101 @@
+# The env for storing rolling_group_by methods
+polars_sql_context__methods <- new.env(parent = emptyenv())
+
+#' @export
+wrap.PlRSQLContext <- function(
+  x,
+  frames = NULL,
+  register_globals = FALSE
+) {
+  self <- new.env(parent = emptyenv())
+  self$`_ctxt` <- x
+  self$frames <- frames
+  self$register_globals <- register_globals
+
+  class(self) <- c("polars_sql_context", "polars_object")
+  self
+}
+
+pl__SQLContext <- function(..., register_globals = FALSE) {
+  wrap({
+    self <- PlRSQLContext$new() |>
+      wrap()
+    self$register_many(...)
+    self
+  })
+}
+
+ensure_lazyframe <- function(obj) {
+  as_polars_lf(obj)
+}
```

Comment:
What is this function needed for?

Reply:
It's basically an alias for as_polars_lf(), I thought you'd like to have it for symmetry with py-polars: https://github.com/pola-rs/polars/blob/d95e343e04d47bcbeafc815a2d9855aa5099f1bb/py-polars/polars/sql/context.py#L471

But we can drop it ofc.

Reply:
After the change due to earlier comment, it's basically an alias for as_polars_lf(), I thought you'd like to have it for symmetry with py-polars: https://github.com/pola-rs/polars/blob/d95e343e04d47bcbeafc815a2d9855aa5099f1bb/py-polars/polars/sql/context.py#L471

But we can drop it ofc.

Reply:
I think this is only necessary because Python does not have a generic function for such a use, and it is not needed here.

Reply:
Done